src/main/java/core/basesyntax/WorkWithFile.java

```
@@ -1,7 +1,49 @@
 package core.basesyntax;
 
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.List;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
+        final String Supply = "supply";
+        final String Buy = "buy";
+        int supplyAmount = 0;
+        int buyAmount = 0;
+
+        File myFile = new File(fromFileName);
+        List<String> lines;
+        try {
+            lines = Files.readAllLines(myFile.toPath());
+        } catch (IOException e) {
+            throw new RuntimeException("Can't read file");
```

This violates checklist item: "Don't ignore exceptions". When catching an IOException, you should rethrow a RuntimeException with the original exception as the cause for better debugging: `throw new RuntimeException("Can't read file", e);`